recipes/default.rb

```diff
@@ -8,13 +8,27 @@
 include_recipe 'apt'
 include_recipe 'java'
 
-link "/etc/alternatives/java" do
-  to "/usr/lib/jvm/java-7-openjdk-amd64/bin/java"
-end
+#link "/etc/alternatives/java" do
+  #to "/usr/lib/jvm/java-7-openjdk-amd64/bin/java"
+#end
+
+#link "/etc/alternatives/javac" do
+  #to "/usr/lib/jvm/java-7-openjdk-amd64/bin/javac"
+#end
+
+include_recipe 'build-essential'
 
-link "/etc/alternatives/javac" do
-  to "/usr/lib/jvm/java-7-openjdk-amd64/bin/javac"
+case node['platform']
+when 'debian', 'ubuntu'
+  package 'ruby-dev' do
```

Comment:
These are required to use the Bluepill  gem. Bluepill is kind of a heavy solution to the startup of the spigot service, but it works well for Java services and requires less configuration than a shell init script. There's certainly an argument to be made that a simple init script would be just fine.